.github/workflows/build.yml

```
@@ -0,0 +1,43 @@
+name: build
+
+on:
+  push:
+      branches:
+        - 'master'
+  pull_request: ~
+
+jobs:
+    test:
+        name: "Test"
```

Maybe add PHP version and Flow version to the name, as otherwise the automatism will just output the matrix values in the order they were specified for the run. Hence right now for the include cases flow and php version are swapped vs the other builds and "(7.3, 7.0)" will soon turn into "(7.3, 7.3)" making it ambiguous.
```suggestion
        name: "Test (PHP ${{ matrix.php-versions }}, Flow ${{ matrix.flow-versions }})"
```